scripts/components/keepfollowing.lua

```diff
@@ -20,12 +20,44 @@
 -- @license MIT
 -- @release 0.22.0-alpha
 ----
-local SDK = require("keepfollowing/sdk/sdk/sdk")
+local SDK = require "keepfollowing/sdk/sdk/sdk"
 
+local _WORMHOLE_TRAVEL_THREAD_ID = "wormhole_travel_thread"
```

Comment:
Since you've asked. Yes, these are sorted alphabetically, so it would be better to move `_WORMHOLE_TRAVEL_THREAD_ID` to the end, right after `_PUSHING_THREAD_ID`.

In general, if you are in doubt, sort stuff alphabetically, when the order doesn't matter.